### PR TITLE
convert ConversationInputBarViewControllerDelegate

### PR DESCRIPTION
**Pull request by billypchan**
## What's new in this PR?

convert `ConversationInputBarViewControllerDelegate` to Swift

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewControllerDelegate.swift [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+@objc protocol ConversationInputBarViewControllerDelegate: NSObjectProtocol {
+    func inputBar(didComposeText text: String?, mentions: [Mention]?, replyingTo message: ZMConversationMessage?)
```
Comment (johnxnguyen) [inline Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewControllerDelegate.swift]: Do text and mentions need to be nullable?

Comment (billypchan) [reply Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewControllerDelegate.swift]: fixed

File: Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewControllerDelegate.swift [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+@objc protocol ConversationInputBarViewControllerDelegate: NSObjectProtocol {
+    func inputBar(didComposeText text: String, mentions: [Mention], replyingTo message: ZMConversationMessage?)
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewControllerDelegate.swift]: Why rename only one of the delegate method?

Comment (billypchan) [reply Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewControllerDelegate.swift]: restore to the old naming

**Pull request merged**